deploy/005-update-governance.ts

```diff
@@ -0,0 +1,93 @@
+import { parseUnits } from "ethers/lib/utils";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { GovernorBravoDelegate } from "../typechain/contracts/Governance/";
+
+const GOVERNOR_PROXY_TESTNET = "0x5573422A1a59385C247ec3a66B93B7C08eC2f8f2";
+const NORMAL_VIP_TIMELOCK_ADDRESS = "0xce10739590001705F7FF231611ba4A48B2820327";
+const XVS_VAULT_TESTNET = "0x9ab56bad2d7631b2a857ccf36d998232a8b82280";
+
+const PROPOSAL_CONFIGS = [
+  // ProposalType.NORMAL
+  {
+    votingDelay: 150,
+    votingPeriod: 150,
+    proposalThreshold: parseUnits("150000", 18),
+  },
+  // ProposalType.FASTTRACK
+  {
+    votingDelay: 100,
+    votingPeriod: 100,
+    proposalThreshold: parseUnits("200000", 18),
+  },
+  // ProposalType.CRITICAL
+  {
+    votingDelay: 50,
+    votingPeriod: 50,
+    proposalThreshold: parseUnits("250000", 18),
+  },
+];
+
+const TIMELOCK_MIN_DELAYS_TESTNET = {
+  NORMAL: 600,
+  FAST_TRACK: 300,
+  CRITICAL: 100,
+};
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
```

Comment:
I think hardhat deploy has flags to only deploy things once. Since this is an upgrade deployment should we use this flag?